src/mixer/the_news.ts

```diff
@@ -51,18 +51,19 @@ const considerDoingTheNews = (getState: () => RootState) => async () => {
   if (state.settings.doTheNews === "always") {
     await actuallyDoTheNews();
   } else if (state.settings.doTheNews === "while_live") {
-    if (
-      state.broadcast.connectionState === "CONNECTED" ||
-      state.broadcast.connectionState === "LIVE"
-    ) {
-      const transition = await broadcastApiRequest<{
-        autoNews: boolean;
-        selSource: number;
-        switchAudioAtMin: number;
-      }>("/nextTransition", "GET", {});
-      if (transition.autoNews) {
-        await actuallyDoTheNews();
-      }
+    switch (state.broadcast.connectionState) {
+      case "CONNECTED":
+      case "GOING_LIVE":
+      case "LIVE":
```

Comment:
Do we need this? LIVE should happen in the same tick as GOING_LIVE, so would this cause an unnecessary AJAX request?